apps/vault/src/vault/core/service/__test__/unit/signing.service.spec.ts

```diff
@@ -332,4 +341,91 @@ describe('SigningService', () => {
 
     expect(deserialized).toEqual(txRequest)
   })
+  describe('buildSignableTransactionRequest', () => {
+    it('should build a signable transaction request for EIP1559 transaction type', () => {
```

Comment:
You can reduce the boilerplate on test description and still be grammatically correct if you think the application DOES or DOES NOT something.


```suggestion
    it('builds a signable transaction request for EIP1559 transaction type', () => {
```

The inverse of it would be "it does not build". 

People can read both sentences out loud and they make perfect sense in English.